src/main/java/com/pi4j/mvc/multicontrollerapp/view/pui/components/SomePUIComponent.java

```diff
@@ -0,0 +1,5 @@
+package com.pi4j.mvc.multicontrollerapp.view.pui.components;
```

Comment:
if we have the structure catalog.componetns, does this catalog and the corresponding folder make any sense?

Reply:
yes, as the students can implement their own components in there. Those come not from the catalog

Reply:
in new commit, i deleted the folder